### ISSUE TITLE
add leetcode question solution 

### ISSUE THREAD

**Issue by gaurav03kr** [SJ-Kumar/hacktoberfest2022#17]
can i add leetcode quesion solution.

**Comment by SJ-Kumar** [comment]
Yes, if it is related to JavaScript. If else, you can refresh your javascript knowledge from js folder and solve few simple array exercise and commit.
